packages/core/docs/guide/authentication.md

```diff
@@ -0,0 +1,94 @@
+# What's the authentication?
+Authentication is the operation of recognizing the user's identify. It allows us to associate incoming requests with an account or a person. To do that, provided credentials are compared with ones in the database.
+
+## Registering a new user
+Creating a user can be done using `register` method provided by `useUser` [composable](https://v3.vuejs.org/guide/composition-api-introduction.html#basics-of-composition-api). It requires different fields per each integration. Most common are email, password, first name, and last name.   
```

Comment:
Not sure if we should provide information that could not be relevant for some integrations, If users will copy it they can get frustrated if it will not work, I suggest abstracting the params to `user` object